CMakeLists.txt

```diff
@@ -87,24 +87,24 @@ SET(THIRD_PARTY_DIR ${CMAKE_SOURCE_DIR}/third_party)
 SET(LIBSHOGUN_SRC_DIR ${CMAKE_SOURCE_DIR}/src/shogun)
 SET(COMMON_MODULAR_SRC_DIR ${CMAKE_SOURCE_DIR}/src/interfaces/modular/)
 
-SET(AVAILABLE_INTERFACES PythonModular;OctaveModular;JavaModular;PerlModular;RubyModular;CSharpModular;RModular;LuaModular;ScalaModular)
-SET(PythonModularDescription "Python")
-SET(OctaveModularDescription "Octave")
-SET(JavaModularDescription "Java")
-SET(PerlModularDescription "Perl")
-SET(RubyModularDescription "Ruby")
-SET(CSharpModularDescription "C#")
-SET(RModularDescription "R")
-SET(LuaModularDescription "Lua")
-SET(ScalaModularDescription "Scala")
+SET(AVAILABLE_INTERFACES Python;Octave;Java;Perl;Ruby;CSharp;R;Lua;Scala)
+SET(PythonDescription "Python")
```

Comment:
Can we please name those "INTERFACE_PYTHON" etc, since we are touching them already.
There is an issue for that somewhere